### ISSUE TITLE
Output not saved

### ISSUE THREAD

**Issue by jyp256** [EmilHvitfeldt/fastTextR#3]
Hi. I was using a previous version of fastTextR and now reinstalled the new version of the package updated (I guess in Oct 2022).
Previously, it used to generate ".txt" file and ".bin" files, but with the current version even with the "save_output" option on, it does not generate these two files in the working directory. I tried changing the specification of the directory in multiple different ways but still no luck.

Below I am attaching my code. Can you please help? Thank you!

`PATH_INPUT = "/Users/myproject/output_token_single_file.txt"
PATH_OUT = "/Users/myproject/rt_fst_model"

vecs <- fastTextR::ft_train(file = PATH_INPUT, method = "skipgram",
control = ft_control(learning_rate = 0.02, learn_update = 100, output = PATH_OUT, save_output = TRUE,
word_vec_size = 400, window_size = 5, epoch = 15, min_count = 1, neg = 5,
max_ngram = 2, loss = "hs", nbuckets = 2e+06,
nthreads = 8, t = 1e-04, verbose = 2))`
